Makefile

```diff
@@ -101,6 +111,9 @@ test-performance-cleanup:
 lint: check-warn ## Lint the project
 	SKIP="no-commit-to-branch" pre-commit run --all-files
 
+deploy: check-deploy-warn ## Deploy the project
+	cd terraform/infrastructure && $(MAKE) ENV=$(ENV) TF_WORKSPACE=$(TF_WORKSPACE) apply
```

Comment:
Have you committed the `apply` target? I can't see it in the Makefile, but I'm not sure if I'm missing something.

Reply:
I didn't scroll down far enough :sweat_smile: